test/testdrive/kafka-exactly-once-sink.td

```diff
@@ -8,6 +8,8 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
+$ set-regex match=\d{13} replacement=<TIMESTAMP>
```

Comment:
Why does this stick around?

Reply:
I think the story is that #14673 reintroduces the tests that need the regex, but hopefully #14678 will remove the need for it again. 🎢 

Either way, no reason for it to be in this PR... an artifact of me flailing around a bit I think.